packages/payments-client/src/utils/fetcher.ts

```diff
@@ -0,0 +1,7 @@
+import { StringMap } from '@reapit/elements'
+import { genPlatformHeaders } from '../utils/headers'
+
+export const fetcher = async (path: string) =>
+  fetch(`https://platform.dev.paas.reapit.cloud${path}`, {
```

Comment:
Please use window.reapit.config for the platform url so it is context aware. 

Also, this fetcher does not catch any errors, can you handle the error correctly please?